modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/exportimport/data/handler/DDMStructureStagedModelDataHandler.java

```diff
@@ -544,6 +549,59 @@ private void _exportDDMFormLayout(
 			ddmFormLayoutPath, structureLayout.getDefinition());
 	}
 
+	private void _exportReferencedDDMStructure(
+		PortletDataContext portletDataContext, DDMStructure structure,
+		DDMFormField ddmFormField) {
```

Comment:
@mbowerman could you please sort this alphabetically?

Reply:
could you also rename from `structure` to `ddmStructure`? Thanks!

Reply:
@carolmariaabb Done